src/main/play-doc/operation/DynamicProxyConfiguration.md

```diff
@@ -0,0 +1,627 @@
+
+## Default configuration
+
+ConductR provides the following HAProxy configuration as a default:
+
+```
+global
+    log /dev/log    local0
+    log /dev/log    local1 notice
+    chroot /var/lib/haproxy
+    stats socket /run/haproxy/admin.sock mode 660 level admin
+    stats timeout 30s
+    user haproxy
+    group haproxy
+    daemon
+
+    # Default SSL material locations
+    ca-base /etc/ssl/certs
+    crt-base /etc/ssl/private
+
+    # Default ciphers to use on SSL-enabled listening sockets.
+    # For more information, see ciphers(1SSL). This list is from:
+    #  https://hynek.me/articles/hardening-your-web-servers-ssl-ciphers/
+    ssl-default-bind-ciphers ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5:!DSS
+    ssl-default-bind-options no-sslv3
+
+defaults
+    log global
+    mode    http
+    option  httplog
+    option  dontlognull
+        timeout connect 5000
+        timeout client  50000
+        timeout server  50000
+    errorfile 400 /etc/haproxy/errors/400.http
+    errorfile 403 /etc/haproxy/errors/403.http
+    errorfile 408 /etc/haproxy/errors/408.http
+    errorfile 500 /etc/haproxy/errors/500.http
+    errorfile 502 /etc/haproxy/errors/502.http
+    errorfile 503 /etc/haproxy/errors/503.http
+    errorfile 504 /etc/haproxy/errors/504.http
+
+{{#eachAcls bundles defaultHttpPort=9000}}
+
+  {{#ifAcl 'conductr-kibana' '1' 'kibana'}}
+# ConductR - Kibana Bundle HAProxy Configuration
+frontend kibana_frontend
+  bind {{haproxyHost}}:5601
+  mode http
+  acl kibana_context_root path_beg /
+  use_backend kibana_backend if kibana_context_root
+
+backend kibana_backend
+  mode http
+    {{#eachBackendServer}}
+  server {{serverName}} {{host}}:{{port}} maxconn 1024
+    {{/eachBackendServer}}
+  {{/ifAcl}}
+
+  {{#ifAcl 'visualizer' '1.1' 'visualizer'}}
+# ConductR - Visualizer Bundle HAProxy Configuration
+frontend visualizer_frontend
+  bind {{haproxyHost}}:9999
+  mode http
+  acl visualizer_context_root path_beg /
+  use_backend visualizer_backend if visualizer_context_root
+
+backend visualizer_backend
+  mode http
+    {{#eachBackendServer}}
+  server {{serverName}} {{host}}:{{port}} maxconn 1024
+    {{/eachBackendServer}}
+  {{/ifAcl}}
+
+{{/eachAcls}}
+
+
+{{#haproxyConf bundles}}
+{{serviceFrontends}}
+{{#unless (serviceFrontends)}}
+frontend dummy
+  bind 127.0.0.1:65535
+{{/unless}}
+{{serviceBackends}}
+{{/haproxyConf}}
+```
+
+The default configuration above is written in [Handlebars](http://handlebarsjs.com/) template, and thus the template makes use of variables and helper functions made available to the template.
+
+### Dynamic proxy configuration DSL
+
+The variables and helper functions exposed to the template forms the DSL required to generate HAProxy configuration from the information available within ConductR cluster.
+
+The following variables are available to the template.
+
+* [bundles](#bundles)
```

Comment:
What about using a table here instead of sub sections for each variable:

| Name              | Type   | Description |
|-------------------|--------|-------------|
| haproxyHost       | String | Required: Host address of the proxy of which the configuration will be generated for.<br><br>Example: 10.0.2.223 |
| bundles | JSON array | Required: List of bundles returned by the ConductR bundles endpoint. This list is in the form of an arrays where each element is a bundle represented by a JSON object. The cluster-wide information of what has been deployed and what is currently running is being stored in this variable.<br><br>Example: See Query bundle state |
| overrideConfigDir | String | Optional: Physical directory location on the file system where configuration overrides has been unpacked. This variable is only defined when HAProxy custom template for ConductR HAProxy has been supplied.<br><br>Example: /tmp/0d24d10cb0d1af9bf9f7e0bf81778a61-e357d11e8a38228880aa57d976d6f83b/e357d11e8a38228880aa57d976d6f83b576fd95e21dc99e8e4d02d4b227b4be3 |

Tested this table locally. The example is part of the description column. Initially I've used a separate column for the examples, but unfortunately the last example `/tmp/0d24d10cb0d1af9bf9f7e0bf81778a61-e357d11e8a38228880aa57d976d6f83b/e357d11e8a38228880aa57d976d6f83b576fd95e21dc99e8e4d02d4b227b4be3` is very long which destroys the column wrapping. So putting the example into the description column looks good.